### PR TITLE
[luci] Introduce node graphlet create helper classes

### PR DESCRIPTION
**Pull request by seanshpark**
This will introduce two helper classes NodeIsGraphletT and
NodeIsGraphletT that will create node graphlet that will provide one
input or arbitrary inputs.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #6596, from draft #6634

File: compiler/luci/partition/src/ConnectNode.test.h [hunk 1]
```diff
@@ -47,6 +47,38 @@ template <unsigned N> class TestIsOGraph : public TestIsGraphlet<N>, public Test
...
+    _node = g->nodes()->create<T>();
+    _node->dtype(loco::DataType::S32);
+    _node->name("node");
+  }
+
+  T *node(void) const { return _node; }
+
+protected:
+  T *_node{nullptr};
+};
```
Comment (mhs4670go) [inline compiler/luci/partition/src/ConnectNode.test.h:64]: I think this could be substituted by `NodeIsGraphletT<1>`.

Comment (seanshpark) [reply compiler/luci/partition/src/ConnectNode.test.h:64]: Yes, you are right. But for reading the code, without `<1>` would be better...

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**